index.js

```diff
@@ -65,9 +73,9 @@ function wrapWord(rows, word, cols) {
 			continue;
 		}
 
-		visible++;
+		visible += charLength;
 
-		if (visible >= cols && i < word.length - 1) {
+		if (visible >= cols && i < arr.length - 1) {
```

Comment:
And this should probably be `visible === cols` to avoid duplicating the `if` statement above.